ci/L0_backend_vllm/metrics_test/vllm_metrics_test.py

```diff
@@ -134,7 +124,7 @@ def test_vllm_metrics(self):
         # vllm:prompt_tokens_total
         self.assertEqual(metrics_dict["vllm:prompt_tokens_total"], 18)
         # vllm:generation_tokens_total
-        self.assertEqual(metrics_dict["vllm:generation_tokens_total"], 188)
+        self.assertEqual(metrics_dict["vllm:generation_tokens_total"], 48)
```

Comment:
Could you please update comment above with the expected answer and explain why we check `48` . I would like to avoid magic numbers in the code, to speed up the debugging time in the future.

Reply:
I didn't find a way to print generation tokens directly from vLLM engine. And the calculation of `generation_tokens_total` isn't straight forward (see https://github.com/vllm-project/vllm/blob/da1f7cc12a12ea4a744d26122e9a13ea4b3f4c7b/vllm/engine/llm_engine.py#L1086-L1088).

Reply:
How did you define `48` as an expected number of tokens ?

Reply:
Assuming vLLM reports the correct number of tokens in the current version (0.5.3-post1), this test makes sure the number stays consistent.

Reply:
Amazing, thanks for this update !